staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -57,6 +57,7 @@ func init() {
 
 	// we need to add the options (like ListOptions) to empty v1
 	metav1.AddToGroupVersion(aggregatorscheme.Scheme, schema.GroupVersion{Group: "", Version: "v1"})
+	metav1.AddToGroupVersion(Scheme, schema.GroupVersion{Group: "", Version: "v1"})
```

Comment:
interesting that this is related to the PR.

Reply:
Schema here duplicated with aggregatorscheme.Scheme. I think keep one is ok.